experimental/dds/tree/.eslintrc.js

```diff
@@ -18,16 +18,6 @@ module.exports = {
 				avoidEscape: true,
 			},
 		],
-		// Intentionally not unifying signatures can enable more scoped API documentation and a better developer experience, which accounts
```

Comment:
These overrides are no longer required with the prettier config pulled in above.